slash/frontend/slash_run.py

```diff
@@ -58,6 +56,14 @@ def slash_run(args, report_stream=None, resume=False, app_callback=None, working
 
 slash_resume = functools.partial(slash_run, resume=True)
 
+def report_tests_to_backslash(tests):
+    active_plugins = manager.get_active_plugins()
+    if 'backslash' not in active_plugins:
```

Comment:
Better to `active_plugins.get('backslash', None)` and then compare to `None`. Prevents duplication